frontend/src/components/ui/chat/Chat.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Hi, if not too troublesome can help remove this import? It's not needed, thanks!